qutip/entropy.py

```diff
@@ -4,7 +4,9 @@
 
 from numpy import conj, e, inf, imag, inner, real, sort, sqrt
 from numpy.lib.scimath import log, log2
-from . import (ptrace, ket2dm, tensor, sigmay, partial_transpose,
+from numpy import np
```

Comment:
please remove this, it is failing the tests (it should be import numpy as np, but np is not used anywhere? so better just to remove)